runtime/autoload/rustfmt.vim

```diff
@@ -61,7 +61,7 @@ function! s:RustfmtWriteMode()
     endif
 endfunction
 
-function! s:RustfmtConfigOptions()
+function! rustfmt#RustfmtConfigOptions()
```

Comment:
all the callers of `s:RustfmtConfigOptions()` should be changed to `rustfmt#RustfmtConfigOptions()`